lighting.py

```diff
@@ -103,6 +105,9 @@ def attributeChangeCallback(
     size: int,
     value: bytes,
 ):
+    if not value:
+        value = bytes([0])
```

Comment:
Please add comments, otherwise it will be hard to understand why this was done later.